js/common.js

```diff
@@ -944,6 +944,29 @@ var templateItilStatus = function(option) {
     return $(`<span><i class="itilstatus ${classes}"></i> ${option.text}</span>`);
 };
 
+var templateTaskStatus = function(option) {
```

Comment:
we use this part only one time.
Imo, an inline script is preferable (in `dropdownState` php function)